application/src/test/java/com/sanction/thunder/email/MessageOptionsConfigurationTest.java

```diff
@@ -0,0 +1,37 @@
+package com.sanction.thunder.email;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.google.common.io.Resources;
+
+import io.dropwizard.configuration.YamlConfigurationFactory;
+import io.dropwizard.jackson.Jackson;
+import io.dropwizard.jersey.validation.Validators;
+
+import java.io.File;
+import javax.validation.Validator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class MessageOptionsConfigurationTest {
+  private final ObjectMapper mapper = Jackson.newObjectMapper();
+  private final Validator validator = Validators.newValidator();
+  private final YamlConfigurationFactory<MessageOptionsConfiguration> factory
+      = new YamlConfigurationFactory<>(MessageOptionsConfiguration.class, validator, mapper, "dw");
+
+  @Test
+  void testFromYaml() throws Exception {
```

Comment:
Can you use assertThrows() here too?

Reply:
These might have already been here but if you can change it it'll be better I think!